apex/normalization/instance_norm.py

```diff
@@ -0,0 +1,808 @@
+from enum import Enum
+from typing import Any, Dict, List, Optional, Tuple
+
+import torch
+from torch.nn.modules.batchnorm import _NormBase
+
+from nvfuser._C import DataType, Fusion, FusionDefinition, Scalar, Tensor
```

Comment:
Sorry for breaking the news, but @kevinstephano had refactored python API and this part likely needs some patching for that.

Reply:
Also, let's now worry about it for now. IIUC, we also need the TOT devel branch for this to work?!

Reply:
> IIUC, we also need the TOT devel branch for this to work?!

yes, I'm comfortable waiting for the refactoring pr on pytorch/pytorch